src/main/java/seedu/weme/ui/CommandBox.java

```diff
@@ -96,7 +101,9 @@ private void handleIndexToggleByKeyPress(KeyEvent event) {
             int change = event.getCode().equals(KeyCode.LEFT) ? -1 : 1;
             int currentLikeCount = Integer.parseInt(argument.trim());
```

Comment:
Should it be named `currentIndex` or smth?

Reply:
Oops, must be overwritten somehow.. It was from the first few commits. Corrected.